acpreprocessing/stitching_modules/convert_to_n5/tiff_to_ngff.py

```diff
@@ -658,23 +927,29 @@ class TiffDirToN5InputParameters(argschema.ArgSchema,
                                  N5GroupGenerationParameters):
     input_dir = argschema.fields.InputDir(required=True)
     out_n5 = argschema.fields.Str(required=True)
-    interleaved_channels = argschema.fields.Int(required=False, deafult=1)
+    output_format = argschema.fields.Str(required=True)
+    interleaved_channels = argschema.fields.Int(required=False, default=1)
     channel = argschema.fields.Int(required=False, default=0)
 
 
 class TiffDirToN5(argschema.ArgSchemaParser):
     default_schema = TiffDirToN5InputParameters
 
     def run(self):
-        tiffdir_to_n5_group(
-            self.args["input_dir"], self.args["out_n5"],
+        tiffdir_to_ngff_group(
+            self.args["input_dir"], self.args["output_format"], 
+            self.args["out_n5"],
             self.args["group_names"], self.args["group_attributes"],
             self.args["max_mip"],
             self.args["mip_dsfactor"],
             self.args["chunk_size"],
             concurrency=self.args["concurrency"],
             compression=self.args["compression"],
-            lvl_to_mip_kwargs=self.args["lvl_to_mip_kwargs"])
+            #lvl_to_mip_kwargs=self.args["lvl_to_mip_kwargs"],
```

Comment:
Can you give a use case where you're seeing failures?  This is an important thing for performance and while we don't have to integrate it with your code in this PR we should not break it in `run`

Reply:
I'm not sure, it might have been a problem with my argschema handling in general, I'll see if I can recreate it